docker-compose.yml

```diff
@@ -45,6 +49,7 @@ services:
       - dev-db
       - test-db  # use -e SQLALCHEMY_TEST_DATABASE_URI=... to exec pytest
       - queue-db
+      - mailhog  # Added MailHog dependency
```

Comment:
Technically, both the web server and the workers send emails, so we can also add the dependency to the worker.

Reply:
okay done.